frontend/src/component/common/DialogFormTemplate/ConfigButtons/DropdownList.styles.tsx

```diff
@@ -1,8 +1,10 @@
 import { Checkbox, ListItem, styled } from '@mui/material';
 
 export const StyledListItem = styled(ListItem)(({ theme }) => ({
-    paddingLeft: theme.spacing(1),
+    paddingLeft: theme.spacing(2),
+    paddingBlock: theme.spacing(1),
```

Comment:
These shouldn't be made here; this is just a placeholder while I work out the PR.